train.py

```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
     model_config.max_seq_len = job_config.training.seq_len
 
     logger.info(f"Building {model_name} {job_config.model.flavor} with {model_config}")
+    logger.info(
```

Comment:
let's move this logging to https://github.com/pytorch/torchtitan/blob/main/train.py#L358